src/11_args.py

```diff
@@ -31,6 +41,11 @@
 
 # YOUR CODE HERE
 
+
+def f3(arg1, arg2=1):
+    return arg1 + arg2
+
```

Comment:
So, this probably wasn't the way they intended this, but I really like the default value solution. :)